parameter.py

```diff
@@ -169,45 +170,7 @@ def deleteAllParameter(methodName:str, className:str):
         return -2
 
     C.classIndex[classIndex].methods[methodIndex].params = []
-
-
-def changeParam(className:str, methodName:str, oldParams:list, newParams:list):
```

Comment:
Is this gone now & we'll call delete and add?

Reply:
I removed it because the original one works for now